tutorials/plot_visualize_evoked.py

```diff
@@ -23,20 +23,21 @@
 ###############################################################################
 # Notice that ``evoked`` is a list of :class:`evoked <mne.Evoked>` instances.
 # You can read only one of the categories by passing the argument ``condition``
-# to :func:`mne.read_evokeds`. To make things more simple for this tutorial, we
-# read each instance to a variable.
-evoked_l_aud = evoked[0]
-evoked_r_aud = evoked[1]
-evoked_l_vis = evoked[2]
-evoked_r_vis = evoked[3]
+# to :func:`mne.read_evokeds`.
+# The read evoked objects can be stored in a dict for easier access.
+conditions = ["Left Auditory", "Right Auditory", "Left visual", "Right visual"]
+evoked = dict()
+for condition in conditions:
+    evoked[condition] = mne.read_evokeds(fname, baseline=(None, 0),
+                                          proj=True, condition=condition)
```

Comment:
I agree with Alex. Let it look like a matlab script to not scare potential
users. Don't show everything Python can do. It's important that people just
get started. Then they'll figure out.
On Mon, 22 Aug 2016 at 09:34, Alexandre Gramfort <notifications@github.com>
wrote:

> In tutorials/plot_visualize_evoked.py
> <https://github.com/mne-tools/mne-python/pull/3528#discussion_r75629265>:
>
> > @@ -23,20 +23,21 @@
> >  ###############################################################################
> >  # Notice that ``evoked`` is a list of :class:`evoked <mne.Evoked>` instances.
> >  # You can read only one of the categories by passing the argument ``condition``
> > -# to :func:`mne.read_evokeds`. To make things more simple for this tutorial, we
> > -# read each instance to a variable.
> > -evoked_l_aud = evoked[0]
> > -evoked_r_aud = evoked[1]
> > -evoked_l_vis = evoked[2]
> > -evoked_r_vis = evoked[3]
> > +# to :func:`mne.read_evokeds`.
> > +# The read evoked objects can be stored in a dict for easier access.
> > +conditions = ["Left Auditory", "Right Auditory", "Left visual", "Right visual"]
> > +evoked = dict()
> > +for condition in conditions:
> > +    evoked[condition] = mne.read_evokeds(fname, baseline=(None, 0),
> > +                                          proj=True, condition=condition)
>
> @jona-sassenhagen <https://github.com/jona-sassenhagen> think about new
> mne users with limited python skills. This is one of the first tutorial
> they will read. I tried to keep it simple (no zip, enumerate) with little
> python kung fu...
>
> I would prefer to keep this tutorial simple and stupid. Makes sense?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/3528/files/d4a3b107affcdfa06c76d4cbdee3a6e795b62d0a#r75629265>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB0finyAqkpoJJAvCChTxAyAzRwhgSVHks5qiVD0gaJpZM4JpRky>
> .
>